### PR TITLE
feat: league_player

### PR DESCRIPTION
**Pull request by sunnyfisch**

### CODE REVIEW

File: dreamscore.yaml [hunk 1]
```diff
@@ -363,11 +363,29 @@ paths:
...
     get:
       tags:
         - /leagues/players
+      summary: Get player information
+      parameters:
+        - $ref: "#/components/parameters/leagueId"
+        - $ref: "#/components/parameters/playerId"
+      responses:
+        "200":
+          description: The information of the player
```
Comment (boostvolt) [inline dreamscore.yaml:388]: missing response content

File: dreamscore.yaml [hunk 2]
```diff
@@ -403,6 +421,16 @@ paths:
     get:
       tags:
         - /leagues/players
+      summary: Get player transfer details
+      parameters:
+        - $ref: "#/components/parameters/playerId"
```
Comment (boostvolt) [inline dreamscore.yaml:426]: leagueId param is missing

File: dreamscore.yaml [hunk 3]
```diff
@@ -650,6 +678,23 @@ components:
...
+          type: string
+          description: The name of the team
+        logoUrl:
+          type: string
+          format: uri
+          description: The URL of the team's logo
+        coverUrl:
+          type: string
+          format: uri
+          description: The URL of the team's cover
```
Comment (boostvolt) [inline dreamscore.yaml:697]: we already have a club schema which should be used (coverUrl is no longer needed)

File: dreamscore.yaml [hunk 4]
```diff
@@ -791,33 +885,23 @@ components:
             $ref: "#/components/schemas/offer"
 
     playerPreview:
+      type: object
```
Comment (boostvolt) [inline dreamscore.yaml:888]: not required. because you defined properties below

File: dreamscore.yaml [hunk 5]
```diff
@@ -791,33 +885,23 @@ components:
...
-          format: int64
-          description: The status of the player (0 - fit, 1 - injured, 2 - stricken)
         position:
           type: integer
           format: int64
           description: The position of the player (0 - goalkeeper - gk, 1 - defense - def, 2 - midfield - mf, 3 - forward - fwd)
-        number:
-          type: integer
-          format: int64
-          description: The number of the player
```
Comment (boostvolt) [inline dreamscore.yaml:820]: we need those infos on the player preview because other endpoints depend on them

File: dreamscore.yaml [hunk 6]
```diff
@@ -403,6 +421,16 @@ paths:
...
+      summary: Get player transfer details
+      parameters:
+        - $ref: "#/components/parameters/playerId"
+      responses:
+        "200":
+          description: The transfer details of the player
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/playerPreview"
```
Comment (boostvolt) [inline dreamscore.yaml:433]: This should return infos visible on the new transfer screen on a player

File: dreamscore.yaml [hunk 7]
```diff
@@ -363,11 +363,29 @@ paths:
...
+      parameters:
+        - $ref: "#/components/parameters/leagueId"
+        - $ref: "#/components/parameters/playerId"
+      responses:
+        "200":
+          description: The details of the player
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/player"
```
Comment (boostvolt) [inline dreamscore.yaml:376]: On this endpoint we should return the player header infos visible in all tabs on a player. it includes portrait, name, club, user, etc.

File: dreamscore.yaml [hunk 8]
```diff
@@ -668,6 +713,55 @@ components:
...
+        averagePoints:
+          type: number
+          format: float
+          description: The average points of the player
+        marketValue:
+          type: number
+          format: float
+          description: The market value of the player
+        marketValueTrend:
+          type: integer
```
Comment (boostvolt) [inline dreamscore.yaml:757]: adjust according the endpoint needs as mentioned above

**Review by boostvolt — CHANGES_REQUESTED** [review]

File: dreamscore.yaml (hunk 5 above)
Comment (sunnyfisch) [reply dreamscore.yaml:820]: But if you need all this information, then you can get the player who has all the information in it.

**Comment by boostvolt** [comment]
closed in favor of #9

**Pull request closed without merging**